src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -114,7 +114,7 @@ if (typeof Sfjs === 'undefined' || typeof Sfjs.loadToolbar === 'undefined') {
 
             var infoSpan = document.querySelector(".sf-toolbar-ajax-info");
             if (infoSpan) {
-                infoSpan.textContent = requestStack.length + ' AJAX request' + (requestStack.length !== 1 ? 's' : '');
+                infoSpan.textContent = requestStack.length + ' AJAX request' + (requestStack.length !== 1 ? 's');
```

Comment:
this change is wrong. This is a JS ternary, not a Twig one

Reply:
Oups... Fixed